pkg/operator/starter.go

```diff
@@ -749,9 +672,7 @@ func withVolumeGroupSnapshotWebhook(enabled bool) dc.DeploymentHookFunc {
 			container := &deployment.Spec.Template.Spec.Containers[i]
 			switch container.Name {
 			case "snapshot-controller":
-				container.Args = append(container.Args, "--enable-volume-group-snapshots")
-			case "webhook":
-				container.Args = append(container.Args, "--enable-volume-group-snapshot-webhook")
+				container.Args = append(container.Args, "-feature-gates=CSIVolumeGroupSnapshot=true")
```

Comment:
can we change `-feature-gates` to `--feature-gates`?

Reply:
fixed.